src/Microsoft.ML.Data/Transforms/Normalizer.cs

```diff
@@ -47,14 +47,16 @@ namespace Microsoft.ML.Transforms
     /// * Log Mean Variance - Rescale each row to unit variance, optionally, zero mean based on computations in log scale.
     /// * Binning - Bucketizes the data in each row and performs a linear rescale based on the calculated bins.
     /// * Supervised Binning - Bucketize the data in each row and performas a linear rescale based on the calculated bins. The bin calculation is based on correlation of the Label column.
+    /// * Robust Scaling - Centers the data and scales based on the range of data and the quantile min and max values provided.
     ///
```

Comment:
suggestion: Append "This method is more robust to outliers"

Reply:
Added.